docs/tutorial.rst

```diff
@@ -804,33 +804,26 @@ in your file system:
 
     (psamm-env) $ psamm-model --model <PATH-TO-MODEL.YAML> fba
 
-By default, PSAMM fba will use the biomass function designated in the central
-model file as the objective function. If the biomass tag is not defined in a
-``model.yaml`` file or if you want to use a different reaction as the
-objective function, you can simply specify what the reaction ID is at the
-end of the command. For example to maximize the citrate synthase reactions,
-CS, the command would be as follows:
-
-.. code-block:: shell
-
-    (psamm-env) $ psamm-model fba CS
-
 The following is a sample of some output from the FBA command:
 
 .. code-block:: shell
 
     INFO: Model: Ecoli_core_model
-    INFO: Model Git version: 9812080
+    INFO: Model Git version: v0.1-819-g4974900-dirty
     INFO: Using Biomass_Ecoli_core_w_GAM as objective
     INFO: Loop removal disabled; spurious loops are allowed
     INFO: Setting feasibility tolerance to 1e-09
-    INFO: Solving took 0.00 seconds
-    ACALDt	0.0	|Acetaldehyde[C_e]| <=> |Acetaldehyde[C_c]|
-    ACKr	0.0	|ATP[C_c]| + |Acetate[C_c]| <=> |Acetyl-phosphate[C_c]| + |ADP[C_c]|
-    ACONTa	6.00724957535	|Citrate[C_c]| <=> |cis-Aconitate[C_c]| + |H2O[C_c]|
-    ACONTb	6.00724957535	|cis-Aconitate[C_c]| + |H2O[C_c]| <=> |Isocitrate[C_c]
+    INFO: Setting optimality tolerance to 1e-09
+    INFO: Solving took 0.05 seconds
+    ACONTa	6.00724957535	|Citrate[c]| <=> |cis-Aconitate[c]| + |H2O[c]|	b0118 or b1276
+    ACONTb	6.00724957535	|cis-Aconitate[c]| + |H2O[c]| <=> |Isocitrate[c]|	b0118 or b1276
+    AKGDH	5.06437566148	|2-Oxoglutarate[c]| + |Coenzyme-A[c]| +
+      |Nicotinamide-adenine-dinucleotide[c]| => |CO2[c]| +
+      |Nicotinamide-adenine-dinucleotide-reduced[c]| + |Succinyl-CoA[c]|	b0116 and b0726 and b0727
```

Comment:
It seems like there is a newline in the copied output. This should be a single line. Did you want to break this long line off instead of showing the whole thing?